scipy/io/netcdf.py

```diff
@@ -987,6 +983,50 @@ def __setitem__(self, index, data):
                 self.data.resize(shape)
         self.data[index] = data
 
+    def _get_missing_value(self):
+        """
+        Returns the value denoting "no data" for this variable.
+
+        If this variable does not have a missing/fill value, returns None.
```

Comment:
Yes, I deliberately reversed the logic that had been in both ```__getitem__``` and ```__setitem__``` in terms of the precedence of ```missing_value``` vs. ```_FillValue```. See my comment below for some additional rationale, in terms of making scipy.io.netcdf more consistent with other popular netCDF tools. However, I'm happy to defer to others in this regard, if there is a reason to keep ```missing_value``` as higher precedence. In practice, this will probably affect few people, because generally if a dataset has both ```_FillValue``` and ```missing_value``` they are identical. But I agree that if this is changed then it should be noted in the release notes.